src/tuning/tuning.cpp

```diff
@@ -119,6 +221,7 @@ void Tuner(int argc, char* argv[], const int V, GetTunerDefaultsFunc GetTunerDef
   args.device_id =
       GetArgument(command_line_args, help, kArgDevice, ConvertArgument(std::getenv("CLBLAST_DEVICE"), size_t{0}));
   args.precision = GetArgument(command_line_args, help, kArgPrecision, Precision::kSingle);
+  args.threads = GetArgument(command_line_args, help, kArgNumThreads, size_t{1});
```

Comment:
Good idea to keep this thread count at 1 as default. Because on some systems, the host and the device might be the same: we don't want parallel compilation to influence the kernel run-time!